services/basic_types.proto

```diff
@@ -1194,7 +1196,10 @@ message NodeAddress {
     AccountID nodeAccountId = 6;
 
     /**
-     * # The hex-encoded SHA-384 hash of the X509 cert used to encrypt gRPC traffic to the node
+     * # Hex encoding of the SHA-384 hash of the TLS cert the node uses to encrypt gRPC 
```

Comment:
This description is bit obtuse and the second sentence duplicates much of the first. Would suggest replacing the first sentence with something that just talks about the cert, keeping the second sentence, and adding a note about what this field can be used for.

```
Every node has a TLS certificate in PEM format that it uses to encrypt gRPC traffic. This field is a string of hexadecimal characters which, translated to binary, are the SHA-384 hash of the UTF-8 NFKD representation of the node's certificate. Its value can be used to verify the node's certificate it presents during TLS negotiations.
```

Reply:
Revised.